src/modules/potions.haml

```diff
@@ -21,54 +24,43 @@
                         %thead
                             %tr
                                 %th Element
+                                %th Attribute
                                 %th Description
-                                %th Value
+                                %th Default
```

Comment:
I still think doing the table this way looks ugly and restricts the length of attribute descriptions (not really a problem in this case).